lambdatimes/src/components/Content/Content.js

```diff
@@ -37,19 +44,28 @@ export default class Content extends Component {
           of the items from cardData. 
         - else, it should only return those cards whose 'tab' matched this.state.selected.
     */
+   if (this.state.selected === 'all') {
     return this.state.cards;
+  } else {
+    return this.state.cards.filter (card => card.tab === this.state.selected);
```

Comment:
IIRC, a space in between the method's properties and the array method itself will not run. I think it would need to look like this:
```suggestion
    return this.state.cards.filter(card => card.tab === this.state.selected);
```

Logic looks fantastic though.